src/components/PlatformNavigator/index.tsx

```diff
@@ -0,0 +1,76 @@
+import { useState } from 'react';
+import { Button, Flex, Text } from '@aws-amplify/ui-react';
+import { IconChevron } from '@/components/Icons';
+import { frameworks } from '@/constants/frameworks';
+import { InfoPopover } from './InfoPopover';
+import Link from 'next/link';
+
+export function PlatformNavigator({ currentPlatform }) {
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+
+  const platformItem = frameworks.filter((platform) => {
+    return platform.title === currentPlatform;
+  })[0];
+
+  return (
+    <>
+      <nav aria-labelledby="platformBtn" className={`platform-navigator`}>
+        <Text fontWeight={"bold"}>Choose your framework/language</Text>
+        <Flex alignItems="center">
+          <Button
+            className={`platform-navigator__button`}
+            aria-expanded="false"
+            id="platformBtn"
+            aria-controls="platformNav"
+            onClick={() => {
```

Comment:
Can we move focus to the navigation element for keyboard navigators? I have an example of that here, in addition to closing the popover when clicking outside of it. https://github.com/aws-amplify/docs/blob/next-release/main/src/components/GetStartedPopover/GetStartedPopover.tsx#L69-L83